cms/djangoapps/contentstore/views/tests/test_item.py

```diff
@@ -26,6 +26,9 @@
     add_container_page_publishing_info
 )
 from contentstore.tests.utils import CourseTestCase
+from lms_xblock.mixin import (
```

Comment:
Or perhaps this does meet isort standards... in which case ignore my comment entirely!